### DISCUSSION TITLE
Universe 2023 tickets are available now! 🎟️

### DISCUSSION THREAD

**Discussion by tuves** [community/community#63834] [Announcements]
https://github.com/community/community/assets/49915749/0f65d42d-ff7f-4d48-9739-da9c501b9f82

GitHub Universe tickets are now available (with an Early-Bird discount!). 

**[Get them here](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)**

### So what is GitHub Universe?
GitHub Universe is our annual global event that brings developers, enterprise leaders, and security professionals together for two days of **networking**, **conversing** with peers and the team at GitHub, and **learning** about the latest innovations. 
This year, we’ll have over 80 sessions with 1,000 speakers covering **AI, security, and DevEx**.

### If I come IRL, what can I expect? 
So much: 

- The Hub - Home to the GitHub booth, our Innovation Hall with our partners, and the Demos & Donuts stage. 
- The Galleries - Immerse yourself in art and technology while you visit our Innovation Hall across the gallery spaces and participate in the Discussion Lounge. 
- Ask GitHub - Get your technical questions answered with our product experts. No pre-booking required! 
- The Shop - Refresh your octocat stash on Level 2 of the Gallery building.
- The Garden - Grab a bite to eat, enjoy the coffee airstream bar, and check out the live stream of our on-site broadcast studio. 
- GitHub certification testing - Just for in-person attendees: the exclusive beta launch of GitHub certifications along with a testing location on site! During the event, you'll have the opportunity to take an exam, allowing you to leave with an official GitHub Certification in Actions, Security, or Administration. 

### When is it? 🗓️

- November 8-9, 2023

### Where is it? 🎨

- IRL at the Yerba Buena Center for the Arts in San Francisco. 
- Virtually from wherever you are. (Streaming from [githubuniverse.com](http://githubuniverse.com/)!) 

### What are the ticket types?

- There are virtual and in-person ticket options and multiple session formats, so you can customize a learning experience that works for you. 

### How can I get my boss to pitch in? 

- Lucky you: We already wrote out [this email you can send right to your boss](https://static.rainfocus.com/github/universe23/static/staticfile/staticfile/Persuade%20Your%20Boss_1691623195585001MLXi.pdf). All you have to do is copy, paste, and fill in the blanks. 

**[Get your Early-Bird tickets](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)**

**Comment by Rodocasa** [comment]
Hello world! Friends? Or no time for it? I have 20 to documented? There's already 10 that I couldn't even see. Good thing the craziness wasn't all alone. I know I don't have a human capable of reading all the DATE in my lifetime, but I'm different not to mention: Wild, lazy and unruly the nicest I've learned. I am realy see mensages from nice people now.

**Comment by Mahmood3214** [comment]
Very good

**Comment by VGandhi27** [comment]
Want to be part of this GitHub Universe

**Comment by Shibukub454** [comment]
![-vjix7d](https://github.com/community/community/assets/133131010/87028af1-2a8b-43ea-913a-680dc694e374)

https://github.com/community/community/assets/133131010/c5403767-5948-44e8-a4d6-c0a3f8839361

**Comment by geoffreylgv** [comment]
My ticket for GH Universe 23 (Virtual)
Hi @tuves, how the virtual mode will be?  

![geoffreylgv GHUuniverse23](https://github.com/community/community/assets/52314615/efa2aa3d-340f-4f68-97f4-62fd87eef10d)

**Comment by RishikeshYadav95** [comment]
Just got my ticket for Universe 2023. Looking forward to meeting the community!
Thank you @tuves 👍🏻

**Comment by ettaboyle** [comment]
### ⏰  Update: **GitHub Universe: Super Early Bird Tickets Ending Soon!**
Early bird tickets are available until September 1st, 2023. Don't delay and secure your spot now! [[Get your Early-Bird tickets here](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)]

**Comment by reeveng** [comment]
When will my beloved homepage be available again? https://github.com/orgs/community/discussions/categories/feed

**Comment by xieyuheng** [comment]
Please bring back the old feed!

**Comment by Md-arefin** [comment]
Thank you so much. 😍

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
